WordPress/Classes/ViewRelated/Blog/SharingButtonsViewController.swift

```diff
@@ -843,6 +819,16 @@ import WordPressShared
     }
 
 
+    override func tableView(tableView: UITableView, canEditRowAtIndexPath indexPath: NSIndexPath) -> Bool {
+        // We only actual editing (ordering) with sections that canSort and are not toggled in the UI activating/deactivating.
```

Comment:
This comment doesn't read properly to me – `we only actual editing (ordering) with sections` – you could clarify please?